site/source/docs/compiling/Building-Projects.rst

```diff
@@ -53,6 +53,29 @@ To build with Emscripten, you would instead use the following commands:
   Some build systems may not properly emit bitcode using the above procedure, and you may see ``is not valid bitcode`` warnings. You can run ``file`` to check if a file contains bitcode (also you can manually check if the contents start with ``BC``). It is also worth running ``emmake make VERBOSE=1`` which will print out the commands it runs - you should see *emcc* being used, and not the native system compiler. If *emcc* is not used, you may need to modify the configure or cmake scripts.
 
 
+.. _building-projects-build-outputs:
+
+Emscripten build output files
+=============================
+
+Emscripten compiler output often consists of several files and not just one. The set of produced files changes depending on the final linker flags passed to `emcc/em++`. Here is a cheat sheet of which files are produced under which conditions:
+
+ - `emcc ... -o output.html` builds a `output.html` file as an output, as well as an accompanying `output.js` launcher file, and a `output.wasm` WebAssembly file.
+ - `emcc ... -o output.js` omits generating a HTML launcher file (expecting you to provide it yourself), and produces two files, `output.js` and `output.wasm`.
```

Comment:
I'd rephrase this as: we just emit js and wasm here. If you run in node.js, that might be enough, or you can write your own HTML to load it in etc.

Reply:
ok, updated.